### PR TITLE
Add Contribution Guide

### PR DESCRIPTION
**Pull request by Skn0tt**
Closes: #127 

This is a first Draft on our new Contribution Guide.

## What happened to the old Contribution Guide?
It wasn't accurate to this project, and it's guidelines were never followed.
I've replaced the branching spec with a reference to the GitHub Flow.

@phlprcks @rohansaw @sspangenberg @Lasklu Please leave feedback: Which parts are still missing? Which parts could be explained in more detail?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,128 +1,90 @@
...
+<!-- /code_chunk_output -->
 
-The main branch should be considered `origin/develop` and will be the main branch where the source code of `HEAD` always reflects a state with the latest delivered development changes for the next release. As a developer, you will be branching and merging from `develop`.
+## Introduction to the Codebase
 
-Consider `origin/master` to always represent the latest code deployed to production. During day to day development, the `master` branch will not be interacted with.
+This introduction assumes that you've used Kalle and are familiar with the problem it solves.
 
-When the source code in the `develop` branch is stable and has been deployed, all of the changes will be merged into `master` and tagged with a release number. _How this is done in detail will be discussed later._
+### Term Definitions
```
Comment (Skn0tt) [inline CONTRIBUTING.md:22]: I'd like to move our term definitions out of the wiki, into here.
Reasons:
1. Colocating Contribution Docs
2. Version the Term Definitions together with the rest of the project

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,128 +1,90 @@
...
 
-The different types of branches we may use are:
+#### Example of finding an Appointment
 
-- Feature branches
-- fix branches
+Alice wants to meet with Bob to talk about Goldfishes. For this purpose, she creates a _Meeting_ in Kalle with the topic "Goldfish Update". She sends Bob the link to this _meeting_ (kalle.app/alice/goldfish-updates).
 
-Each of these branches have a specific purpose and are bound to strict rules as to which branches may be their originating branch and which branches must be their merge targets. Each branch and its usage is explained below.
+Bob opens the link and gets a lot of _time slots_ displayed in wich Alice is available. He chooses one of them. With this he creates an _Appointment Suggestion_.
```
Comment (rohansaw) [inline CONTRIBUTING.md:44]: We acutally don't have suggestions, we only book appointments directly without them being confirmed

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,128 +1,90 @@
...
-- Must branch from: `develop`
-- Must merge back into: `develop`
-- Branch naming convention: `feature/GH-<tbd number>`
+`npm install` to fetch dependencies.
+`npm start` to start the dev environment.
+<kbd>CTRL+C</kbd> to stop.
+`docker-compose down` to tear it down.
 
-#### Working with a feature branch
+`npm run db:seed` seeds the database (optional, but recommended).
```
Comment (Lasklu) [inline CONTRIBUTING.md:60]: I'm not quite sure but shouldn't we mention how to merge the db? I think it is required to do so when someone wants to develop?

Comment (sspangenberg) [reply CONTRIBUTING.md:60]: We should definitely add the blitz prisma migrate dev command here or a reference to the corresponding blitz documentation

**Review by phlprcks — APPROVED** [review]
@Skn0tt please also check my changes

**Comment by rohansaw** [comment]
Maybe we should also add a section on how to get started as a developer? So how we strucutre backend and frontend in our code

**Comment by Skn0tt** [comment]
There's a brief section on folder structure, but some more guidance would definitely come in handy. @rohansaw do you want to write some? If not, it'd be handy to have a list of specific questions / topics that need more clarification.

**Comment by rohansaw** [comment]
@Skn0tt jup will add that

**Review by rohansaw — APPROVED** [review]

**Review by sspangenberg — APPROVED** [review]

**Pull request merged**